### PR TITLE
Feature/12219 testprocess family members/12303 user selection

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
PR to add test owner selection screen

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12303

## Screenshots
Simulator Screen Shot - iPhone 13 - 2022-03-22 at 15 05 11
Simulator Screen Shot - iPhone 13 - 2022-03-24 at 13 55 59

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

**Comment by KaiTeuber** [comment]
I'm not sure if dark mode looks correct - I was wondering if it is as requested. The section tiles are not visible in the dark mode.

**Comment by naveeddotio** [comment]
> I'm not sure if dark mode looks correct - I was wondering if it is as requested. The section tiles are not visible in the dark mode.

I will have a look

**Comment by KaiTeuber** [comment]
Dynamic type is broken too

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestOwnerSelection/ExposureSubmissionTestOwnerSelectionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+		dynamicTableViewModel = viewModel.dynamicTableViewModel
+		tableView.separatorStyle = .none
+	}
+}
+
+enum TestOwner {
+	case user
+	case familyMember
+}
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestOwnerSelection/ExposureSubmissionTestOwnerSelectionViewController.swift:65]:
```suggestion
```
Move it over to the viewModel

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestOwnerSelection/ExposureSubmissionTestOwnerSelectionViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	// MARK: - Init
+
+	init(
+		onTestOwnerSelection: @escaping(TestOwner) -> Void
+	) {
+		self.onTestOwnerSelection = onTestOwnerSelection
+	}
+	
+	// MARK: - Internal
+
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/TestOwnerSelection/ExposureSubmissionTestOwnerSelectionViewModel.swift:19]:
```suggestion
enum TestOwner {
	case user
	case familyMember
}
```

**Review by KaiTeuber — CHANGES_REQUESTED** [review]
Dark mode and dynamic type need a bit more work

**Review by KaiTeuber — APPROVED** [review]

**Comment by naveeddotio** [comment]
> Dark mode and dynamic type need a bit more work

Fixed. Thanks for looking into it.

**Review by 30mar — APPROVED** [review]

**Pull request merged**